base/abstractset.jl

```diff
@@ -145,13 +165,26 @@ intersect!(s::AbstractSet, itr) =
 Construct the set of elements in `s` but not in any of the iterables in `itrs`.
 Maintain order with arrays.
 
+Difference of set `A` and `B` is a set that contains the elements that are in `A`, but not in `B`.
```

Comment:
Isn't this pretty much exactly what the first sentence in the docstring say? Also, the word "Difference" seems to not occur anywhere else here which makes it unclear what it refers to.

Reply:
Is the new one better?
https://github.com/JuliaLang/julia/pull/33936/files#diff-69a0054b980f1b009ae561651ad29c54R165

Because the difference is usually a thing between two things, I see this explanation useful.